packages/tailwindcss/src/utilities.ts

```diff
@@ -185,7 +186,7 @@ export function withAlpha(value: string, alpha: string): string {
 /**
  * Apply opacity to a color using `color-mix`.
  */
-export function replaceAlpha(value: string, alpha: string | null): string {
+export function replaceAlpha(value: string, alpha: string): string {
```

Comment:
This type was there on purpose (see check immediately after)

Reply:
Ah yep but im doing the test in the caller now. Can remove the check afterwards now!